lib/auth/auth.go

```diff
@@ -2505,12 +2512,18 @@ func (a *Server) AugmentWebSessionCertificates(ctx context.Context, opts *Augmen
 		return trace.Wrap(err)
 	}
 
+	// We consider this SSH key to be verified because we take it directly from
+	// the web session. The user doesn't need to verify they own it because the
+	// don't: we own it.
+	const sshKeyVerified bool = true
```

Comment:
```suggestion
	const sshKeyVerified = true
```

Reply:
Just a reminder that this suggestion was not changed yet.

Reply:
oh thanks, it was resolved and then i did a bad force-push, made the change again